### ISSUE TITLE
Why is cancelling the master password input considered an error?

### ISSUE THREAD

**Issue by AtmanActive** [iiordanov/remote-desktop-clients#560]
Hi,

Thanks again for creating bVNC Pro for Android. It is really getting better and better all the time.

I've noticed the master-password feature added and I started using it straight away. Good call. However, when I'm done with my session, bVNC goes back to asking for the master password again, which I don't mind, but, since I only want to exit the app at that point, I naturally click cancel. But that click gets me the error message for which I then have to click OK. Very annoying.
In other words, the master-password feature improved the app security and startup experience but it ruined the app exit experience.

In my humble opinion, for this and all other scenarios, clicking on the Cancel button on the master-password input should just exit the app silently and that's it. That error message is completely useless and annoying.

Thanks.

**Comment by iiordanov** [comment]
@AtmanActive I changed the behavior of the app not to request the master password on disconnection. This is no longer an issue, right? Please comment if I should reopen.
